frontend/talentsearch/resources/js/components/search/SearchForm.tsx

```diff
@@ -141,14 +141,13 @@ export const SearchForm: React.FunctionComponent<SearchFormProps> = ({
 
   // Whenever form values change (with some debounce allowance), call updateCandidateFilter
   const formValues = watch();
-  const submitDebounced = useCallback(
+  const submitDebounced = useCallback(() => {
```

Comment:
This was required by an obscure rule. Apparently useCallback need an inline function:
https://reactjs.org/docs/hooks-reference.html#usecallback

Reply:
I'm actually having issues with this one, specifically where it's called below context in L155 and L159

Can anyone advise on how debouncing should work here? It's new to me :)

The examples of can find for addressing the rule use [`debounce-fn`](https://github.com/sindresorhus/debounce-fn) pkg, and we use [`debouce`](https://github.com/component/debounce)

cc @tristan-orourke as author